File: algorithm/src/main/java/baekjoon/p02525/Main.java
package baekjoon.p02525;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();

        int totalMinute = (a * 60) + b + c;
        int hour = (totalMinute / 60) % 24;
        int minute = totalMinute % 60;

        System.out.print(hour + " " + minute);
    }

}

File: algorithm/src/main/java/baekjoon/p02525/Main.kt
package baekjoon.p02525

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val a = sc.nextInt()
    val b = sc.nextInt()
    val c = sc.nextInt()

    val totalMinute = (a * 60) + b + c
    val hour = (totalMinute / 60) % 24
    val minute = totalMinute % 60

    println("$hour $minute")
}